src/client/components/measures-list/measures-list.vue

```diff
@@ -25,22 +49,34 @@
 </template>
 
 <script>
-import { mapState } from 'vuex'
 
 export default {
+  props: {
+    measures: {
+      type: Array,
+      default: () => []
+    },
+    parcelsPerMeasure: {
+      type: Object,
+      default: () => ({})
+    }
+  },
   data() {
     return {
       selectedMeasure: {},
```

Comment:
Why is no `selectedMeasure` not simply `undefined`? Easier to have a `selectedMeasureId`?